OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -117,7 +117,7 @@ public static void ProcessOrder(OrderManager orderManager, World world, int clie
 						if (client != null)
 						{
 							var pause = order.TargetString == "Pause";
-							if (orderManager.World.Paused != pause && !world.LobbyInfo.IsSinglePlayer)
+							if (orderManager.World.Paused != pause && world != null && !world.LobbyInfo.IsSinglePlayer)
```

Comment:
> you're mixing `orderManager.World` and `world`.

Not me, this was actually introduced by yourself in https://github.com/OpenRA/OpenRA/pull/2991 so I assume you knew what you were doing back then. If you spotted a mistake of your own, you need to elaborate.